docs/guide/file_management.md

```diff
@@ -101,6 +101,21 @@ The way to configure other adapters can be found in the [bundle's online documen
 
 For clustering use we provide a custom metadata handler that stores metadata about your assets in the database. This is done as it is faster than accessing the remote NFS or S3 instance in order to read metadata. For further reading on setting this up, see [Clustering](clustering.md).
 
+## Enable BinaryFile Field indexing
+
+The indexing of all binary file fields is disabled by default. If you want to enable indexing, you must have installed Oracle Java/Open JDK 8 or higher and Apache Tika 1.20.
+Next, in the `config/packages/` create a `<name>.yaml` file with the following configuration:
+
+``` yaml
+ibexa_platform_commerce_field_types:
+  binary_file_indexing:
+    enabled: true
+```
+
+To check what file types are indexed, see under the `siso_search.<site_access>.index_content` parameter in `src/Siso/Bundle/SearchBundle/Resources/config/search.yml`. 
```

Comment:
Those file types are indexed by default (in `siso_search.default.index_content`), that param can be overriden, if you want to extend indexing to other file types, or be used to index only specific types per SA. Like in, `en` SA you want to index only pdf files for some reason.